src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -80,6 +80,16 @@ class CoordinateMapBase : public PUP::able {
                             SpatialIndex<Dim, UpLo::Lo, SourceFrame>>>
   jacobian(const tnsr::I<DataVector, Dim, SourceFrame>& source_point) const = 0;
   // @}
+ protected:
```

Comment:
I thought we had decided we didn't want comparison for abstract base classes? This could be implemented as a helper function for the tests, then we'd only require `operator==` for the derived classes. @wthrowe I recall us discussing this issue.

Reply:
Why make it difficult to test the code?  I do not always know the derived class when testing.

Reply:
If you don't know what you're testing you shouldn't be testing it. It also doesn't make it difficult to test, it just moves the functionality out of the class. It's actually less code to write a test outside of the base class.

Reply:
I do know what I am testing (that the serialization of Block and Domain are done correctly) and this makes it much easier.

Reply:
I see what you're saying. Okay, well we should discuss with Will this week, see what his thoughts are.

Reply:
I don't like having it, but I recognize Larry's point that this is necessary for writing sane serialization tests for domain stuff, so I think we're stuck with it.

On a different note, is there a reason it has to be protected rather than private?